components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -313,11 +313,15 @@ export class ReportingComponent implements OnInit, OnDestroy {
     const queryParams = {...this.route.snapshot.queryParams};
     const endDate = moment.utc(event.detail);
 
-    if (moment().utc().format('YYYY-MM-DD') === moment(endDate).format('YYYY-MM-DD')) {
-      delete queryParams['end_time'];
```

Comment:
No longer need to remove the end_time in this case...?

Reply:
Nope. Empty end_time query param is now used for 'last 24hr' instead of 'current day'.